tests/e2e/systemtest/test_noobaa_system.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.bucket_utils import (
+    write_data_to_buckets_and_check_integrity,
+    verify_s3_buckets_integrity,
+)
+
+log = logging.getLogger(__name__)
+
+
+class TestNoobaaSystem(E2ETest):
```

Comment:
Could we change this to TestNoobaaBackupRecoverySystemTest?

Reply:
Also, I think it'd be right to refactor the name to MCG throughout all files. 
Most of our code uses `MCG` and not NooBaa - MCG being the correct downstream terminology.